src/style.css

```diff
@@ -1 +1,127 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 600;
+}
+
+:root {
+  --main-color: #3d4e61;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  color: var(--main-color);
+  font-weight: 300;
+}
+
+body {
+  /* padding: 0; */
+  box-sizing: border-box;
+}
+
+.search {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+  box-sizing: border-box;
+}
+
+.search--big {
+  width: 100%;
+  height: 70px;
+}
+
+.search--small {
+  width: 100%;
+  height: 42px;
+}
+
+.search__label {
+  width: 100%;
+  position: relative;
+  display: block;
+}
+
+.search__label::after {
+  content: '';
+  position: absolute;
+  display: block;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+}
+
+.search__label--big::after {
+  top: 25px;
+  left: 26px;
+  width: 19px;
+  height: 19px;
+}
+
+.search__label--small::after {
+  top: 15px;
+  left: 13px;
+  width: 11px;
+  height: 11px;
+}
```

Comment:
Be mindful of hardcoding pixel values for positioning. If the design is responsive, consider using relative units or CSS Flexbox or Grid for more flexible layouts.